packages/engine/src/worker/mod.rs

```diff
@@ -64,13 +69,16 @@ pub struct WorkerController {
     py: PythonRunner,
     js: JavaScriptRunner,
     rs: RustRunner,
-    _config: WorkerConfig, // TODO: unused, remove?
+    // TODO: unused, remove?
+    _config: WorkerConfig,
     worker_pool_comms: WorkerCommsWithWorkerPool,
     tasks: PendingWorkerTasks,
 }
 
 // TODO: impl drop for worker controller?
 impl WorkerController {
+    /// Spawns a new worker for each language: JavaScript, Python, and Rust and initialize the
```

Comment:
Each worker has one of each language runners (depending on if they're enabled), rather than there being a worker _per_ language runner